### PR TITLE
README: swath of updates to reflect the current state

### PR DESCRIPTION
**Pull request by cardoe**

### CODE REVIEW

**Comment by cardoe** [comment]
These comments I believe are true for krogoth, morty and master. If approved I'll backport the change.

**Comment by cardoe** [comment]
Obviously a README change didn't break the build but I added another test slave to the mix and it exposed an assumption in the test scripts. I'll add a commit to this branch to fix it.

**Comment by srwalter** [comment]
:+1:

**Comment by cardoe** [comment]
ok seriously now fix the issue with the new builder added

File: scripts/fetch.sh [hunk 1]
```diff
@@ -40,11 +40,11 @@ update_repo() {
 		git fetch origin || die "unable to fetch ${uri}"
 	else
 		echo "Cloning '${path}'"
-		if [ -z "${GIT_LOCAL_REF_DIR}" ]; then
-			git clone ${uri} ${path} || die "unable to clone ${uri}"
-		else
+		if [ -d "${GIT_LOCAL_REF_DIR}" ]; then
 			git clone --reference ${GIT_LOCAL_REF_DIR}/`basename ${path}` \
                 ${uri} ${path} || die "unable to clone ${uri}"
```
Comment (derekstraka) [inline scripts/fetch.sh]: The whitespace is a little funny here

Comment (cardoe) [inline scripts/fetch.sh]: Fixed the whitespace in the latest push

File: scripts/fetch.sh [hunk 2]
```diff
@@ -40,7 +40,7 @@ update_repo() {
 		git fetch origin || die "unable to fetch ${uri}"
 	else
 		echo "Cloning '${path}'"
-		if [ -z "${GIT_LOCAL_REF_DIR}" ]; then
+		if [ -z "${GIT_LOCAL_REF_DIR}" -a ! -d "${GIT_LOCAL_REF_DIR}" ]; then
```
Comment (cardoe) [inline scripts/fetch.sh]: yeah I dropped the whole check cause it didn't really matter. But yeah there it needed to be an or.

**Comment by cardoe** [comment]
Backported into krogoth.

**Comment by cardoe** [comment]
Pushed to morty now too.

**Pull request merged**